creditcoin-js/README.md

```diff
@@ -77,6 +80,49 @@ const batch_tx = api.tx.utility.batch(txs);
 await batch_tx.signAndSend(alice);
 ```
 
+### Registering External Addresses
+```typescript
+import { personalSignSignature } from 'creditcoin-js/lib/extrinsics/register-address-v2';
+import { personalSignAccountId } from 'creditcoin-js/lib/utils';
+import { Wallet } from "creditcoin-js";
```

Comment:
Can we add this as files into the `examples/` directory? In the past we tried executing them in CI to make sure they still work for example https://github.com/gluwa/creditcoin/blob/dev/integration-tests/src/test/loan-cycle.test.ts

Reply:
I see examples have been added but README still contains big chunks of code, which people will copy over. 

My intention was that code blocks inside README will be replaced with something like
"see [src/examples/collect-coins-v2.ts](https://github.com/gluwa/creditcoin/pull/1346/files#diff-10bdd3dbb06404a823408f0d179124e6afb0cbb4605480be42d766cf1062d695)" for anything which is bigger than 5 lines.